client/components/TestQueueRow/TestQueueRow.css

```diff
@@ -77,6 +77,10 @@ button.more-actions:active {
     width: initial;
 }
 
-/* tr.test-queue-run-row td:first-child {
-    padding: 0.75rem;
-} */
+.primary-test-run-select {
+    padding: 0 !important;
```

Comment:
Can we add a padding left to the select?

Reply:
I realize now what was being shown is because the dialog still appears when the number of testers to choose from is 1 and `<select>.size` would render differently.

I've pushed https://github.com/w3c/aria-at-app/pull/1001/commits/b738bc05a30fcee945642a0cae3ac2b6fd431315 which should won't display the dialog at all if there's only 1 tester to choose from. Thanks for pointing this out!